project/app/api/predict.py

```diff
@@ -82,4 +82,34 @@ async def covid_by_state(state: dict):
     # filter df for above info and get the total
     new= df[(df['state'] == state) & (df['submission_date'] > last_week)]
     new_cases= new['new_case'].astype('float').sum()
-    return new_cases
\ No newline at end of file
+    return new_cases
+
+@router.get('/fuel')
```

Comment:
Hey Amy... I think I missed this when we talked about this route, I think we need to add a path parameter to the route so we can pass the state abbreviation to the `get_gas_price_state` method

So something like:

`@router.get('/fuel/{ste}`

Reply:
Thanks Dan!!  I think the @router.get is the URL it runs on.  So for this example, in FASTAPI - 'http://901683bb5dd4.ngrok.io/fuel'.  The end can be anything we want it to say.  @router.get('/AnythingIwantToSay') -  And it will still work.  I am not sure changing the path parameter will make a difference??  I could be wrong though!  The '''def get_gas_price_state(ste):''' is what matters.